cypress/integration/calculator/functionTest.spec.js

```diff
@@ -0,0 +1,153 @@
+import {INITAIL_VALUE, ALERT_MSG_OVER_THREE_NUMBER, ALERT_MSG_WITH_OUT_NUMBER} from '../../../src/js/constant.js'
+
+describe('My first test', () => {
+    let randomNum;
+    before('함수 생성', ()=>{
+        randomNum = ()=> {
+            return Math.floor(Math.random() * 9);
+        }
+    })
+
+    beforeEach('매번 index.html에 접속',() => {
+        cy.visit('/index.html')
+    })
+
+    it('숫자 1개 클릭시 total에 1개 숫자가 표시되어야 한다.', () => {
+        let number1 = randomNum();
+        cy.get('.digit').contains(number1).click();
+        cy.get('#total').should('have.text', number1);
+    });
+
+    it('숫자 2개 클릭시 total에 2자리 숫자가 표시되어야 한다.', () => {
+        let numbers = [];
+        numbers.push('1');
+        numbers.push('9');
+        numbers.forEach(no => {
+            cy.get('.digit').contains(no).click();
+        });
```

Comment:
이 부분은 비어있는 `numbers`변수를 만들고 숫자들을 push하는 작업없이 다음과 같이 작성해보면 어떨까요?
```suggestion
        [1, 9].forEach(no => {
            cy.get('.digit').contains(no).click();
        });
```

Reply:
오 좋은꿀팁입니다. 감사합니다!! 바로 적용했어요ㅎ